### PR TITLE
Add Oak Containers binaries to the provenance workflows

### PR DESCRIPTION
**Pull request by andrisaar**

### CODE REVIEW

File: buildconfigs/image_tar.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This is the static build configuration that we use with the docker-based SLSA3 generator for
+# building the `stage0` binary, and its provenance.
```
Comment (rbehjati) [inline buildconfigs/image_tar.toml:2]:
```suggestion
# building the `image_tar` binary, and its provenance.
```

File: buildconfigs/vmlinux.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# This is the static build configuration that we use with the docker-based SLSA3 generator for
+# building the `stage0` binary, and its provenance.
```
Comment (rbehjati) [inline buildconfigs/vmlinux.toml:2]:
```suggestion
# building the `vmlinux` binary, and its provenance.
```

File: buildconfigs/stage1_cpio.toml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# This is the static build configuration that we use with the docker-based SLSA3 generator for
+# building the `stage0` binary, and its provenance.
```
Comment (rbehjati) [inline buildconfigs/stage1_cpio.toml:2]:
```suggestion
# building the `stage1_cpio` binary, and its provenance.
```

**Review by rbehjati — APPROVED** [review]
Thanks. I've set the `provenance:force-run` label on this to see if it works pre-merge.

**Review by mariaschett — APPROVED** [review]
LGTM, great!

**Pull request closed without merging**